test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -0,0 +1,107 @@
+package seedu.addressbook.commands;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalTestPersons;
+
+/**
+ * Test class for the Find Command
+ */
+public class FindCommandTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private AddressBook addressBook;
+    private AddressBook unmutatedAddressBook;
+
+    @Before public void setUp() {
+        this.addressBook = new TypicalTestPersons().getTypicalAddressBook();
+        this.unmutatedAddressBook = new TypicalTestPersons().getTypicalAddressBook();
+    }
+
+    @Test
+    public void execute_find_matchesOnlyIfCaseSensitive() throws IllegalValueException {
+        List<ReadOnlyPerson> expectedOne = TestUtil.createList(new Person(TypicalTestPersons.amy));
+        List<ReadOnlyPerson> expectedZero = Collections.emptyList();
```

Comment:
You can use the same variable `expected` instead of declaring two variables with somewhat 'odd' names. :-)